code/modules/clothing/suits/armor.dm

```diff
@@ -250,6 +246,32 @@
 	src.item_state = "reactiveoff"
 	..()
 
+/obj/item/clothing/suit/armor/reactive/hit_reaction(mob/living/carbon/human/owner, attack_text, final_block_chance)

```

Comment:
I'd use TG's more recent version of this minus the radiation and exempting space turfs.

No one uses the reactive armor on our codebase.